common-ui/src/main/java/org/simple/clinic/common/ui/theme/SimpleColors.kt

```diff
@@ -8,10 +8,15 @@ import androidx.compose.ui.graphics.Color
 
 @Immutable
 data class SimpleColors(
-  val toolbarPrimary: Color = Color.Unspecified,
-  val toolbarPrimaryVariant: Color = Color.Unspecified,
-  val onToolbarPrimary: Color = Color.Unspecified,
-  val material: Colors = lightColors(),
+    val toolbarPrimary: Color = Color.Unspecified,
+    val toolbarPrimaryVariant: Color = Color.Unspecified,
+    val onToolbarPrimary: Color = Color.Unspecified,
+    val material: Colors = lightColors(),
+    val veryLowRiskColor: Color = Color(0xFF00B849),
+    val lowRiskColor: Color = Color(0xFFFFC800),
+    val mediumRiskColor: Color = Color(0xFFFF5252),
+    val highRiskColor: Color = Color(0xFFB81631),
+    val veryHighRiskColor: Color = Color(0xFF731814),
```

Comment:
We can merge this in for now, but ideally we don't place these at `Theme` level. Since these tokens are specific to statin nudge rather than for entire theme. We will refactor this later when we are removing Compose theme adapter and adding native Compose theme.